open-api/rest-catalog-open-api.yaml

```diff
@@ -2763,7 +2763,8 @@ components:
         
         ## General Configurations
         
-        - `token`: Authorization bearer token to use for table requests if OAuth2 security is enabled 
+        - `token`: Authorization bearer token to use for table requests if OAuth2 security is enabled
+        - `expires-at-ms`: if present, specifies timestamp in milliseconds when credentials for storage(AWS S3/Azure/), specified in config would expire
```

Comment:
```suggestion
        - `expires-at-ms`: if present, specifies timestamp in milliseconds when credentials for storage(AWS S3/Azure/GCP), specified in config would expire
```

Reply:
Also, can we make it clear that it's about credentials? Maybe `credentials-expire-at-ms`? Because as it is, the reader would naively think it's the table itself that expires, or even worse, that it's the `token` above that expires.

Reply:
I was thinking to keep the `config` properties aligned with #10722 , will revisit this once the credential standardisation is finalised